kit/ChildSession.cpp

```diff
@@ -539,10 +539,10 @@ bool ChildSession::uploadSignedDocument(const char* buffer, int length, const St
         Poco::Net::SSLManager::instance().initializeClient(consoleClientHandler, invalidClientCertHandler, sslClientContext);
 
         std::unique_ptr<Poco::Net::HTTPClientSession> psession;
-        psession.reset(new Poco::Net::HTTPSClientSession(
+        psession = Util::make_unique<Poco::Net::HTTPSClientSession>(
```

Comment:
We can assign directly...

        std::unique_ptr<Poco::Net::HTTPClientSession> psession
            = Util::make_unique<Poco::Net::HTTPSClientSession>(

I used clang-format to avoid long lines, but you can do the same manually, if you don't have clang-format.

Reply:
I've updated the patch @Ashod. It seems iOS check failed again.

Reply:
Thanks @bayramcicek.